### PR TITLE
Add preglobal Subaru Forester and Outback

### PR DESCRIPTION
**Pull request by martinl**
This PR adds dashcam only support for pre-global platform Subaru Forester and Outback. Forester and 2018-19 Outback have reversed driver torque signals and require https://github.com/commaai/panda/pull/564 for additional panda safetyParam

### CODE REVIEW

**Comment by martinl** [comment]
Tests failed due to missing routes

File: selfdrive/test/test_car_models.py [hunk 1]
```diff
@@ -389,10 +389,18 @@ def get_route_log(route_name):
...
+    'enableCamera': True,
+  },
   "df5ca7660000fba8|2020-06-16--17-37-19": {
     'carFingerprint': SUBARU.LEGACY_PREGLOBAL,
     'enableCamera': True,
   },
+  "5ab784f361e19b78|2020-06-08--16-30-41": {
+    'carFingerprint': SUBARU.OUTBACK_PREGLOBAL,
+    'enableCamera': True,
+  },
```
Comment (adeebshihadeh) [inline selfdrive/test/test_car_models.py:403]: Let's add a comment that these are routes from the dashcam merge. Once we bring these out of dashcam, I'd like to get new routes.

File: selfdrive/test/test_car_models.py [hunk 2]
```diff
@@ -449,6 +457,9 @@ def get_route_log(route_name):
   TOYOTA.HIGHLANDER,
   TOYOTA.HIGHLANDERH,
   TOYOTA.HIGHLANDERH_TSS2,
+  SUBARU.FORESTER_PREGLOBAL,
+  SUBARU.OUTBACK_PREGLOBAL,
+  SUBARU.OUTBACK_PREGLOBAL_2018,
```
Comment (adeebshihadeh) [inline selfdrive/test/test_car_models.py:462]: no need to add these here for dashcam only cars

File: selfdrive/car/subaru/values.py [hunk 3]
```diff
@@ -8,21 +8,26 @@ class CAR:
...
+  FORESTER_PREGLOBAL = "SUBARU FORESTER 2017 - 2018"
   LEGACY_PREGLOBAL = "SUBARU LEGACY 2015 - 2018"
+  OUTBACK_PREGLOBAL = "SUBARU OUTBACK 2015 - 2017"
+  OUTBACK_PREGLOBAL_2018 = "SUBARU OUTBACK 2018 - 2019"
 
 FINGERPRINTS = {
   CAR.ASCENT: [{
   # SUBARU ASCENT LIMITED 2019
     2: 8, 64: 8, 65: 8, 72: 8, 73: 8, 280: 8, 281: 8, 290: 8, 312: 8, 313: 8, 314: 8, 315: 8, 316: 8, 326: 8, 544: 8, 545: 8, 546: 8, 552: 8, 554: 8, 557: 8, 576: 8, 577: 8, 722: 8, 801: 8, 802: 8, 805: 8, 808: 8, 811: 8, 816: 8, 826: 8, 837: 8, 838: 8, 839: 8, 842: 8, 912: 8, 915: 8, 940: 8, 1614: 8, 1617: 8, 1632: 8, 1650: 8, 1657: 8, 1658: 8, 1677: 8, 1722: 8, 1743: 8, 1759: 8, 1785: 5, 1786: 5, 1787: 5, 1788: 8
+  # END ASCENT
```
Comment (adeebshihadeh) [inline selfdrive/car/subaru/values.py:20]: remove these end comments

File: selfdrive/test/test_car_models.py [hunk 4]
```diff
@@ -389,10 +389,18 @@ def get_route_log(route_name):
     'carFingerprint': SUBARU.IMPREZA,
     'enableCamera': True,
   },
+  "95441c38ae8c130e_2020-06-08--12-10-17": {
```
Comment (adeebshihadeh) [inline selfdrive/test/test_car_models.py:392]: pipe, not underscore

File: selfdrive/test/test_car_models.py (hunk 1 above)
Comment (martinl) [reply selfdrive/test/test_car_models.py:403]: ok, done

File: selfdrive/car/subaru/values.py (hunk 3 above)
Comment (martinl) [reply selfdrive/car/subaru/values.py:20]: done

File: selfdrive/test/test_car_models.py (hunk 4 above)
Comment (martinl) [reply selfdrive/test/test_car_models.py:392]: ok

File: selfdrive/test/test_car_models.py (hunk 2 above)
Comment (martinl) [reply selfdrive/test/test_car_models.py:462]: ok

Comment (martinl) [reply selfdrive/test/test_car_models.py]: SUBARU.OUTBACK_PREGLOBAL_2018 was needed since it does not have a route

**Comment by adeebshihadeh** [comment]
LGTM, will merge after adding a 2018 Outback route and tests pass.

**Comment by martinl** [comment]
Added the route for Outback 2018. Also added the same route to panda safety PR, replacing preglobal Forester route

**Pull request merged**